### PR TITLE
add tanjunchen to sig-docs-zh team

### PR DESCRIPTION
**Pull request by tanjunchen**
I want to join the sig-docs-zh team: 

I've become reviewer for zh docs.
I have worked for website for a long time.
So far, I have participated in website pr(460+) and issue (15+).
I hope I can have more permissions to make more contributions to the zh localization of the website.
https://github.com/kubernetes/website/pulls?q=is%3Apr+assignee%3Atanjunchen+is%3Aclosed   196 
https://github.com/kubernetes/website/pulls?q=is%3Apr+author%3Atanjunchen+is%3Aclosed  106
https://github.com/kubernetes/website/pulls?q=is%3Apr+reviewed-by%3A%40me+    132
https://github.com/kubernetes/website/issues?q=is%3Aopen+is%3Aissue++tanjunchen+  16 
such as https://github.com/kubernetes/website/issues/17649、https://github.com/kubernetes/website/issues/18411

/cc @tengqm

### CODE REVIEW

**Comment by tengqm** [comment]
LGTM
It would be great to have you in the team, @tanjunchen

File: config/kubernetes/sig-docs/teams.yaml [hunk 1]
```diff
@@ -342,6 +344,7 @@ teams:
     - SataQiu # L10n: Chinese
     - seokho-son # L10n: Korean
     - sftim # L10n: English
+    - tanjunchen # L10n: Chinese
```
Comment (zacharysarah) [inline config/kubernetes/sig-docs/teams.yaml:347]: Please remove this line. Enough L10n team members have write access.

**Review by zacharysarah — CHANGES_REQUESTED** [review]
@tanjunchen One change, otherwise LGTM

File: config/kubernetes/sig-docs/teams.yaml (hunk 1 above)
Comment (tanjunchen) [reply config/kubernetes/sig-docs/teams.yaml]: @zacharysarah  done.

**Pull request merged**